Duplicati/Library/Backend/AzureBlob/AzureBlobBackend.cs

```diff
@@ -210,6 +290,16 @@ public void CreateFolder()
         public void Dispose()
         {
 
-        }
+        }
+
+        private static Stream StringToStream(string str)
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream) { AutoFlush = true };
+            writer.Write(str);
+            stream.Position = 0;
+            return stream;
+        }

```

Comment:
Similar to in pull request #4548, can we remove this method and move these statements to the single use case above?

Reply:
Definitely. Just wanted to get feedback on #4548 . Thanks for the other feedback above.